testplan/testing/multitest/result.py

```diff
@@ -143,11 +168,13 @@ def wrapper(result, *args, **kwargs):
         finally:
             if top_assertion:
                 assertion_state.in_progress = False
+                assertion_state.filepath = None
+                assertion_state.line_range = None
```

Comment:
cleaning this in an assertion will make it not working for any more assertion. If you have a call chain where two assertion is called from a non marked helper which is called from a marked one, the first assertion will be reported properly into the marked helper, but the second one will be reported to the non marked one.